src/deploy-web.js

```diff
@@ -46,13 +46,13 @@ const deployWeb = async (config, log) => {
     await getTvmCredentials(config.ow.namespace, config.ow.auth, config.s3.tvmUrl, config.s3.credsCacheFile)
 
   const remoteStorage = new RemoteStorage(creds)
-  const exists = await remoteStorage.folderExists(config.s3.folder)
+  const exists = await remoteStorage.folderExists(config.s3.folder + '/')
```

Comment:
why do you need the backslash ?

Reply:
This is because of the s3 stage bucket permissions.  `folder` fails with `AccessDenied` but `folder/` works.
The creds have permission to the specific folder. We are passing the folder name as `prefix` to listObjectsV2. So it looks like `folder/` means listing the folder itself whereas `folder` means listing the parent folder.

Reply:
ok sweet